### ISSUE TITLE
Rewrite scratch pad Mathematical Answer Examples to use labels

### ISSUE THREAD

**Issue by dqnykamp** [Doenet/DoenetApps#2995]
The scratch pad document Mathematical Answer Examples frequently uses `<shortDescription>` to address accessibility. However, we decided that it is easier for people to use and understand `<label>`. Most of the examples should be rewritten to use `<label>` instead.
